### PR TITLE
Add link to installation instructions to getting started guide

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
Fixes https://github.com/holoviz/panel/issues/1924

I've added the line below

image

I cannot remove Python 2.7 mention as the section below is not included in the notebook or Panel repository.

image

### CODE REVIEW

**Comment by philippjfr** [comment]
Thanks @MarcSkovMadsen, the rest of the contents can be found here https://www.github.com/holoviz/panel/tree/master/doc%2Fgetting_started%2Findex.rst

**Comment by MarcSkovMadsen** [comment]
Thanks. Next step is on me to finalize

**Comment by philippjfr** [comment]
Any chance we can get this in 0.10.4? Hoping to release by tomorrow. If you won't get a chance I'll try to finish this off.

**Comment by MarcSkovMadsen** [comment]
Done @philippjfr

File: examples/getting_started/Introduction.ipynb [hunk 1]
```diff
@@ -409,9 +411,22 @@
...
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
    "name": "python",
-   "pygments_lexer": "ipython3"
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.4"
```
Comment (philippjfr) [inline examples/getting_started/Introduction.ipynb:429]: Please avoid committing notebook metadata.

**Pull request merged**